mrmustard/physics/ansatze.py

```diff
@@ -66,7 +66,7 @@ def __init__(self) -> None:
         self._kwargs = {}
 
     @abstractmethod
-    def from_function(cls, fn: Callable, **kwargs: Any) -> Ansatz:
+    def from_function(cls, fn: Callable, **kwargs: any) -> Ansatz:
```

Comment:
shouldn't `any` be `Any`?

Reply:
Actually yes! You're right. It's not until now that I found out `any` is referring to the function and not the type